models/app-models.js

```diff
@@ -44,4 +42,37 @@ const fetchReviewsbyId = (req) => {
     });
 };
 
-module.exports = { fetchCategories, fetchReviews, fetchReviewsbyId };
+const fetchReviewIdComments = (req) => {
+  const { params } = req;
+  return db
+    .query(
+      `
+      SELECT comments.*, 
+      CAST(COUNT(comment_id) AS INTEGER) AS comment_count 
+      FROM reviews 
+      LEFT JOIN comments ON comments.review_id = reviews.review_id 
+      WHERE reviews.review_id = $1
+      GROUP BY comments.comment_id 
```

Comment:
do you need to join the reviews in this query? doesnt the comments table have. review id column?